includes/models/class-listing.php

```diff
@@ -670,6 +670,20 @@ public function get_fee_plan() {
         return $res;
     }
 
+	/**
+	 * Get the fee plan id.
+	 *
+	 * @since x.x
+	 *
+	 * @return int|bool The plan id or false.
+	 */
+	public function get_fee_plan_id() {
+		global $wpdb;
+
+		$sql = $wpdb->prepare( "SELECT fee_id FROM {$wpdb->prefix}wpbdp_listings WHERE listing_id = %d LIMIT 1", $this->id );
```

Comment:
This should go through our DB functions with a caching layer.

Reply:
Code removed as its not being used